src/restic/cmd/restic/cmd_snapshots.go

```diff
@@ -119,13 +119,7 @@ func (cmd CmdSnapshots) Execute(args []string) error {
 			continue
 		}
 		id := sn.ID()
-		tab.Rows = append(tab.Rows, []interface{}{hex.EncodeToString(id[:plen/2]), sn.Time.Format(TimeFormat), sn.Hostname, sn.Paths[0]})
-
-		if len(sn.Paths) > 1 {
```

Comment:
With 80 character default terminal width it was unreadable to have all fields visible.